en/ros2/user_guide.md

```diff
@@ -751,6 +751,117 @@ will generate topics under the namespaces:
 
 :::
 
+## PX4 ROS 2 Service Servers
```

Comment:
OK, so I have modified this to have the structure
- Intro 
  - what a ros 2 service is and why is it cool
  - List of services in PX4
 - VehicleCommand service
   - Example
 - More services as they come along ....
 
 This is where we centralise service docs.